Changelog.md

```diff
@@ -18,6 +18,7 @@ Compiler Features:
 
 
 Bugfixes:
+ * ABI decoder: Always create copies of dynamic memory arrays during decoding.
```

Comment:
Hmm could we add the motivation for this as well? What exactly is fixed by this

Reply:
I extended the entry - not sure how I feel about very long Changelog entries, though :-)... but do you think it's better now?

Reply:
It's a bit long indeed :laughing: How about something like `Don't unexpectedly modify original array during decoding`?

Reply:
That's not what happens, though :-).

Reply:
What about now :-)?